setup.py

```diff
@@ -4,66 +4,61 @@
 setup.py
 A module that installs projectname as a module
 """
-from glob import glob
-from os.path import basename, splitext
 from pathlib import Path
 
 from setuptools import find_packages, setup
 
 #: Load version from source file
 version = {}
-with open('src/projectname/version.py') as fp:
-    exec(fp.read(), version)
+version_file = Path(__file__).parent / "src" / "projectname" / "version.py"
+exec(version_file.read_text(), version)
 
 setup(
-    name='projectname',
-    version=version['__version__'],
-    license='MIT',
-    description='Project description.',
-    long_description=(Path(__file__).parent / 'README.md').read_text(),
-    long_description_content_type='text/markdown',
-    author='UGRC',
-    author_email='ugrc-developers@utah.gov',
-    url='https://github.com/agrc/python',
-    packages=find_packages('src'),
-    package_dir={'': 'src'},
-    py_modules=[splitext(basename(path))[0] for path in glob('src/*.py')],
+    name="projectname",
+    version=version["__version__"],
+    license="MIT",
+    description="Project description.",
+    long_description=(Path(__file__).parent / "README.md").read_text(),
+    long_description_content_type="text/markdown",
+    author="UGRC",
+    author_email="ugrc-developers@utah.gov",
+    url="https://github.com/agrc/python",
+    packages=find_packages("src"),
+    package_dir={"": "src"},
     include_package_data=True,
     zip_safe=True,
     classifiers=[
         # complete classifier list: http://pypi.python.org/pypi?%3Aaction=list_classifiers
-        'Development Status :: 5 - Production/Stable',
-        'Intended Audience :: Developers',
-        'Topic :: Utilities',
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: MIT License",
+        "Topic :: Utilities",
     ],
     project_urls={
-        'Issue Tracker': 'https://github.com/agrc/python/issues',
+        "Issue Tracker": "https://github.com/agrc/python/issues",
     },
-    keywords=['gis'],
+    keywords=["gis"],
     install_requires=[
-        # 'package==1.0.*'
+        # 'package=1.0.*'
     ],
     extras_require={
-        'tests': [
-            'pylint-quotes~=0.2',
-            'pylint~=2.11',
-            'pytest-cov~=4.0',
-            'pytest-instafail~=0.4',
-            'pytest-isort>=2,<4',
-            'pytest-pylint~=0.18',
-            'pytest-watch~=4.2',
-            'pytest>=6,<8',
-            'yapf~=0.31',
+        "tests": [
+            "pytest-cov==4.*",
```

Comment:
Did you want to switch to some other standard for specifying dependency versions or did we decide that it doesn't matter?

Reply:
it didn't matter